database/src/migrations/20211207105001_remove_user_roles.ts

```diff
@@ -0,0 +1,20 @@
+import * as Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+
+  set schema '${DB_SCHEMA}';
+  set search_path = ${DB_SCHEMA},public;
+
+  DELETE FROM system_role WHERE name = 'Government User';
+  DELETE FROM system_role WHERE name = 'External User';
+  DELETE FROM system_role WHERE name = 'Public User';
+
```

Comment:
Do we need to re-build the views after deleting these records?

Reply:
The two views impacted are the IDIR request form and the complement that assigns roles. Those two are fine since they read from full list.  I can't think of other views that are impacted atm.